itest/send_test.go

```diff
@@ -250,6 +256,13 @@ func testBasicSendPassiveAsset(t *harnessTest) {
 	secondAsset := rpcAssets[1]
 	genInfo2 := secondAsset.AssetGenesis
 
+	testVectors := &proof.TestVectors{}
+	addProofTestVectorFromFile(
```

Comment:
Rationale for this is that we're able to have more complete coverage as these vectors are generated e2e vs with canned data? Trying to understand what the gap here was with the existing unit test based vectors.

Reply:
The issue is that I created some of the test vectors (for example `proof/testdata/proof_tlv_encoding_other.json`) by manually taking some proofs from an integration test run to add some "real" proofs that actually fully validate and aren't just unit test random/dummy data.
I usually updated those files manually again when something in the proof format changed. But that became a bit tedious and wasn't really documented anywhere. So now this process is automated.